media-fonts/ibm_plex/ibm_plex-0.5.3.recipe

```diff
@@ -0,0 +1,34 @@
+SUMMARY="The package of IBM’s new typeface, IBM Plex"
+DESCRIPTION="Thanks for your interest in IBM Plex, IBM’s new corporate type family. \
+We welcome your feedback and contributions to this project. \
+IBM Plex is open source, and should be used by IBMers for all typographical situations, \
+replacing Helvetica Neue, whenever possible."
+HOMEPAGE="https://ibm.github.io/type"
+COPYRIGHT="2017 IBM Corp. with Reserved Font Name 'Plex'"
+LICENSE="SIL Open Font License v1.1"
+REVISION="1"
+SOURCE_URI="https://github.com/IBM/type/archive/v$portVersion.tar.gz"
+CHECKSUM_SHA256="0e212de6d0f33604b6c22e9743aab05bc12d482b1bdc7402972f3f6a9aa92a77"
+SOURCE_DIR="type-$portVersion"
+
+ARCHITECTURES="any"
+DISABLE_SOURCE_PACKAGE=yes
+
+PROVIDES="ibm_plex = $portVersion"
+REQUIRES=""
+BUILD_REQUIRES=""
+BUILD_PREREQUIRES="coreutils"
```

Comment:
There's no need to specify coreutils as `BUILD_PREREQUIRES`

```diff
-BUILD_PREREQUIRES="coreutils"
+BUILD_PREREQUIRES=""

Reply:
I **_could_** say that we **_could_** consider this as an exception, and consider that since `BUILD_REQUIRES` is empty, haikuporter **_could_** infer that it does not need to implicitly add coreutils to `BUILD_PREREQUIRES`, so we would have to add it ourselves.. but this time I agree with @alaviss.

Wait, I found one recipe which also has `BUILD_PREREQUIRES="coreutils"`.
So I'd say we can keep it.

Reply:
It is rarely a good idea to depend directly on a package. If a specific command is needed, it should be listed explicitly.

This is especially important for coreutils, because it is GPL licenced and some days we may switch to an implementation with an MIT licence (for example extracted from one of the BSDs), or switch to another implementation of some of the tools for variuous reasons, or just split the coreutils package in smaller parts.

So, if there is a dependency on a specific command, use the appropriate cmd: entry. Otherwise, jsut leave this empty.

Reply:
Oops, sorry @pulkomandy. I only noticed your comment about the coreutils package after I had merged the PR. 
Well, there were already 43 recipes in the media-font category, out of 56, which explicitly have  coreutils in their `BUILD_PREREQUIRES`.
If needed, I can fix my error (and all the others too) in a bulk update, and using a PR, but this won't be possible before the end of the GCI...

Reply:
Well there is no emergency in doing this (as we don't currently plan to move away from coreutils). It's a cleanup we can do later when there is less activity :)